draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.stories.tsx

```diff
@@ -32,17 +32,17 @@ export default {
 }
 
 export const SingleCollapsibleKaizenSiteDemo = () => (
-  <div style={{ margin: "1rem", width: "40rem" }}>
+  <Box m={1}>
```

Comment:
If you have time, it would be good if this were a decorator instead of having to do it on each story individually.

Reply:
I don't have particularly _strong_ feelings about the best way to do this, but if we move this to be a decorator it won't be displayed in the code sample (when you click "show code") in Storybook. Mostly this is okay, but if we're modifying attributes like `display` we're giving users a false sense of what the component takes care of.

Reply:
For margins it feels like a decorator is okay, but I wonder if it's a pattern we should seldom use, to stop this issue: https://github.com/cultureamp/kaizen-design-system/issues/1750